.github/ISSUE_TEMPLATE/new-trove-classifier.md

```diff
@@ -0,0 +1,11 @@
+Request to add a new Trove classifier.
+
+## The name of the classifier you would like to add
+<!-- Classifier name -->
+
+## Why do you want to add this classifier?
+<!--
+    Please include a brief explanation to justify your request.
+    Why do the current classifiers not meet your need?
+    How many packages to you expect to use this new classifier?
```

Comment:
Hey, I'm a little unclear -- do you mean how many different PyPI *projects*? I figure usually that's what we want to ask, but I am open to being told differently!

Reply:
Nah, project is the right terminology. You would think that I would be better about this. :)

Fixed in ba7560f8